0.9 Release Post (#166)
flowforge/website: src/blog/2022/09/flowforge-09-released.md

---
title: FlowForge 0.9 released
subtitle: Suspended projects, login with email and Team Types
description: Suspended projects, login with email and Team Types
date: 2022-09-01 12:00:00.0
authors: ["sam-machin"]

> video: BBNzFInc_2E

Reviewer: ```suggestion
video: d23Pmyc0k7I
```

Reviewer: The video is currently unlisted - will need to be made public when we publish this post.

---

> Suspend your projects when you don't need them, forgot your username? login with your email instead, and introducing Team Types

Reviewer: ```suggestion
Suspend your projects when you don't need them, login with either your username or email, and introducing Team Types
```

<!--more-->

> Keep reading for the details of whats in this release our you can watch our 1 minute roundup video of the new release above. 

Reviewer: ```suggestion
Keep reading for the details of what's in this release our you can watch our 1 minute roundup video of the new release above. 
```

We're pleased to announce version 0.9 is now available. The next release of the FlowForge application containing new features, a number of improvements, and bug fixes. Keep reading for a promotion code to get your first month free on FlowForge Cloud. 

## Features
[Suspend Projects](https://github.com/flowforge/flowforge/issues/893)

> Sometimes you want to put a project to one side for a while, maybe your development has stalled or you're waiting on something external to be ready. Perhaps you don't need it to be running all the time. With the 0.8 release we've made some changed to how projects are run. You can now suspend a project, what this means is that the container in which your project runs is removed and nothing is running, Your flows  are safely stored in our database but the project isn't consuming any resources. This means that in FlowForge Cloud we also suspend billing you for it.

Reviewer: ```suggestion
Sometimes you want to put a project to one side for a while, maybe your development has stalled or you're waiting on something external to be ready. Perhaps you don't need it to be running all the time. With the 0.9 release we've added the ability to suspend a project. Once suspended, your flows are safely stored in the platform database, but Node-RED isn't running and the project doesn't consume any resources. In FlowForge Cloud we do not charge you for suspended projects - you only pay when the project is running.
```

> Your project will be there ready to start back up when you need it with just one click, the name and hence the URL are still reserved for you.

Reviewer: ```suggestion
Your project will be there ready to start back up when you need it with just one click.
```

Remember that any context data or anything written to the filesystem will not persist through a restart or a suspend of a project.

> As a result of this change we've also removed the facility to stop a project, Stopping only halted the Node-RED process it didn't remove the resources so was of very limited value, as of Node-RED 3.0 you can stop the flows from executing while still being able to make changes which is a better experience.

Reviewer: ```suggestion

Alongside this change, we've removed the option to 'stop' a project. That option would only stop Node-RED, but the underlying container would still be running, consuming resources. With Node-RED 3.0 adding the ability to stop the flows, but still be able to edit them, that provides a much better user experience.
```

You can still restart the Node-RED process from the Forge app as before for example when you have updated a package in your flows.

[Team Types](https://github.com/flowforge/flowforge/issues/733)

> We've introduced another concept into the platform with this release. Team Types will allow us to offer more advanced features to teams on FlowForge Cloud, You won't see much difference in this release but it allows us to build on in future releases.

Reviewer: ```suggestion
We've introduced another concept into the platform with this release. Team Types will allow us to offer more advanced features to teams on FlowForge Cloud. You won't see much difference in this release but it allows us to build on in future releases.
```

[PostHog Analytics](https://github.com/flowforge/flowforge/issues/695)

> We've added analytics to the platform, we're using [PostHog](https://posthog.com/) for this as they share our ethos of open source and self hosting, If you're running your own platform.

Reviewer: ```suggestion
We've changed the analytics tooling integrated into the platform. With this release, we've deprecated the use of Plausible Analytics as it didn't quite provide the sort of insight we wanted. We now integrate with [PostHog](https://posthog.com/). They share our ethos and approach to open source and self hosting - something you can take advantage of if you're running your own FlowForge platform.
```

> For FlowForge Cloud users the data is sent to our PostHog account, but if you're running your own instance then it won't send analytics to us, you are free to configure your own PostHog service though to receive the data yourself.

Reviewer: ```suggestion
For FlowForge Cloud, the data is sent to our PostHog account so we can better understand how the platform is being used. If you're running your own instance, the information is only captured if you configure it with your own PostHog instance details - it does not send any data back to us.
```

[Login with email](https://github.com/flowforge/flowforge/issues/856)

> A common problem that we've seen from users is trying to login with their email address instead of their username, as of 0.8 you can now enter either at the login screen.

Reviewer: ```suggestion
A common problem that we've seen from users is trying to login with their email address instead of their username. As of 0.9 you can now enter either at the login screen.
```

[Custom Dashboard Path](https://github.com/flowforge/flowforge/issues/774)

> You can now change the path where the dashboard will be served from, the default is still /ui but you can now move that onto / or anything else. Helpful when migrating existing projects over to FlowForge.

Reviewer: ```suggestion
If you are using the Node-RED Dashboard set of nodes, you can now change the path where the dashboard will be served from. The default is still `/ui` but you can now move that onto `/` or anything else. This is helpful when migrating existing projects over to FlowForge.
```

## Improvements
We've made a number of improvements to the overall experience of running FlowForge.

- Improvements to the FlowForge Theme [#883](https://github.com/flowforge/flowforge/pull/883). 
- Upper-case characters in Project Names [#546](https://github.com/flowforge/flowforge/issues/546)

> - Accept an update to T&Cs[#451](https://github.com/flowforge/flowforge/issues/451)

Reviewer: ```suggestion
```

Not in this release
